### DISCUSSION TITLE
Can't upload any Files on selfhosted

### DISCUSSION THREAD

**Discussion by timdreier** [outline/outline#6522] [Self Hosting]
Can't upload any Files.

**To Reproduce**
1. Go to any page
2. upload image

**Expected behavior**
Image gets uploaded

**Screenshots**

<img width="386" alt="image" src="https://github.com/outline/outline/assets/10117610/b9817d52-6c9f-4224-aa3e-23624e6f5fd6">

Response from `/api/files.create`:
<img width="393" alt="image" src="https://github.com/outline/outline/assets/10117610/022a99ec-67bf-4ec2-9a90-dfa0ae6e8b49">

**Outline (please complete the following information):**
- Install: self hosted
- Version: e4d007835

**Desktop (please complete the following information):**
 - OS: ubuntu 22
 - Browser Firefox
 - Version 122.0.1

My .env:
```
NODE_ENV=production
SECRET_KEY=xxx
UTILS_SECRET=xxx
DATABASE_URL=postgres://xxx:xxx@xxx.xxx.xxx.xxx/outline
DATABASE_CONNECTION_POOL_MIN=
DATABASE_CONNECTION_POOL_MAX=
REDIS_URL=redis://localhost:637
URL=https://outline.xxx.de
PORT=3000
COLLABORATION_URL=
FILE_STORAGE=local
FILE_STORAGE_LOCAL_ROOT_DIR=/home/xxx/htdocs/data
FILE_STORAGE_UPLOAD_MAX_SIZE=26214400
OIDC_CLIENT_ID=xxx
OIDC_CLIENT_SECRET=xxx
OIDC_AUTH_URI=xxx
OIDC_TOKEN_URI=xxx
OIDC_USERINFO_URI=xxx
OIDC_USERNAME_CLAIM=preferred_username
OIDC_DISPLAY_NAME=Nextcloud
OIDC_SCOPES=openid profile email
SMTP_HOST=
SMTP_PORT=
SMTP_USERNAME=
SMTP_PASSWORD=
SMTP_FROM_EMAIL=hello@example.com
SMTP_REPLY_EMAIL=hello@example.com
SMTP_TLS_CIPHERS=
SMTP_SECURE=true
DEFAULT_LANGUAGE=en_US
RATE_LIMITER_ENABLED=true
RATE_LIMITER_REQUESTS=1000
RATE_LIMITER_DURATION_WINDOW=60
DEVELOPMENT_UNSAFE_INLINE_CSP=false
```

**Comment by tommoor** [comment]
What does the invalid `key` param in the request look like?

**Comment by timdreier** [comment]
@tommoor you mean this?

```
name=\"key\"\r\n\r\nuploads/68002911-0964-4bb8-8e42-e42a75c90afd/3d9b4d25-04e5-4c1d-864e-17ac40d64791//test.png\r\n
```

**Reply by tommoor** [reply]
There are two slashes in the key so it makes sense that this would not pass the validation. Are you using the docker image?

**Comment by timdreier** [comment]
No, installation from source. But where can I configure this?

**Reply by tommoor** [reply]
There is no configuration for this. This bug could make sense, but only if you're using a different commit to the one you specified – there was a bug in the source for about a week on `main` that had similar effects, but that was within the last couple of weeks not back in December.

**Comment by timdreier** [comment]
Just updated to 0.75.0 with exactly the same error :/
